includes/class-wc-stripe-webhook-handler.php

```diff
@@ -600,12 +600,12 @@ public function process_payment_intent_success( $notification ) {
 
 		$order_id = WC_Stripe_Helper::is_wc_lt( '3.0' ) ? $order->id : $order->get_id();
 		if ( 'payment_intent.succeeded' === $notification->type || 'payment_intent.amount_capturable_updated' === $notification->type ) {
-			if ( 'processing' === $order->get_status() || 'completed' === $order->get_status() ) {
+			if ( 'pending' !== $order->get_status() && 'failed' !== $order->get_status() ) {
 				return;
 			}
 
 			$charge = end( $intent->charges->data );
-			WC_Stripe_Logger::log( "Stripe PaymentIntent $intent->id succeeded for order $order_id" );
+			WC_Stripe_Logger::log( "Stripe PaymentIntent $intent->id succeeded for order $order_id from" );
```

Comment:
I think this `from` shouldn't be here.
```suggestion
			WC_Stripe_Logger::log( "Stripe PaymentIntent $intent->id succeeded for order $order_id" );
```

Reply:
The text included some more details at one point, and I thought I removed them completely, but obviously I didn't see a small bit of it. 👍